data_steward/cdr_cleaner/cleaning_rules/drop_orphaned_pids.py

```diff
@@ -0,0 +1,173 @@
+"""
+sandbox and drop participants from the person table if they do not have records in any of the
+AoU required OMOP tables except person.
+
+required tables:
+condition_occurrence, death, device_exposure, drug_exposure, measurement, note,
+observation, procedure_occurrence, specimen, visit_occurrence, visit_detail
+
+"""
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from common import JINJA_ENV, PERSON
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBERS = ['DC-2481']
+
+SANDBOX_ORPHANED_PIDS = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE
+  `{{project}}.{{sandbox_dataset}}.{{intermediary_table}}` AS
+SELECT
+  DISTINCT person_id
+FROM
+  `{{project}}.{{dataset}}.{{person_table}}` p
+LEFT JOIN ( SELECT DISTINCT person_id FROM `{{project}}.{{dataset}}.condition_occurrence`) co
```

Comment:
nit: Would you mind removing the whitespace before `SELECT` to align with the rest of this statement?
From:
```LEFT JOIN ( SELECT ```
To:
```LEFT JOIN (SELECT ```